libs/blocks/merch/merch.js

```diff
@@ -759,6 +759,10 @@ export default async function init(el) {
   const isCta = searchParams.get('type') === 'checkoutUrl';
   const merch = await (isCta ? buildCta : buildPrice)(el, searchParams);
   const service = await initService();
+  if (searchParams.get('type') === 'price') {
+    const a11yParent = el.closest('.col, merch-card, p, div');
```

Comment:
isn't `p` the most comment use case?  IMO the most common one should be at the first place in the selector

Reply:
I was thinking P should be more as backup but I think you're right.

Reply:
Actually, this is kinda pointless.
When this is built. the element isn't fully built. Even if I am checking .col or merch-card, they aren't there yet and their classes aren't ready either. Therefore their nodeName will be always either P or DIV. Which works for most cases but this isn't definitely a bullet proof code.

I wrote a test code already but I think I have to revisit this some hours later.